apps/frontend/src/pages/access-management/access-management.tsx

```diff
@@ -0,0 +1,37 @@
+import { formatDate, type UserAuthResponseDto } from "@git-fit/shared";
+
+import { Table } from "~/libs/components/components.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+} from "~/libs/hooks/hooks.js";
+import { actions as userActions } from "~/modules/users/users.js";
+
+import { TABLE_COLUMNS as headers } from "./libs/constants/constants.js";
+import { type User } from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+const AccessManagement = (): JSX.Element => {
+	const dispatch = useAppDispatch();
+	const users = useAppSelector(({ users }) => users.users);
+
+	useEffect(() => {
+		void dispatch(userActions.loadAll());
+	}, [dispatch]);
+
+	const columns: UserAuthResponseDto[] = users.map((user) => ({
+		...user,
+		createdAt: formatDate(new Date(user.createdAt), "d MMM yyyy HH:mm"),
```

Comment:
we should do this not in data, but in a place where we define columns. We should be able to pass a function to the accessor field in columns. Also instead of having TABLE_COLUMNS constant, let's have getUserColumns helper (in case we need to change columns them dynamically based on params)

Reply:
I don't understand, since an array of complete user objects is returned from the backend, but for this user table we only need the fields "Name", "Group" and "Created At" (for my task only "Name" and "Created At"). Why then can't we store column names in constants if they are not variables?

Do you think it should look something like this?
```
import { formatDate, type UserAuthResponseDto } from "@git-fit/shared";

export const getUserColumns = () => {
return [
 {
 Header: "Name",
 accessor: "name",
 },
 {
 Header: "Email",
 accessor: "email",
 },
 {
 Header: "Created At",
 accessor: (user: UserAuthResponseDto) =>
 formatDate(new Date(user.createdAt), "d MMM yyyy HH:mm"),
 },
 ];
};
```